scripts/dss-start-stop--think-before-you-invoke-me.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""
+Use the AWS Lambda concurrency property to disable/enable lambda functions:
+    - set concurrency to 0 to disable execution
+    - remove concurrency setting to enable execution
+
+As a consequence of this script, previously set Lambda concurrency limits
+will be lost.
+"""
+import os
+import sys
+import boto3
+import click
+import argparse
+
+
+stage = os.environ['DSS_DEPLOYMENT_STAGE']
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("action", choices=["start", "stop"])
+args = parser.parse_args()
+
+
+action = f"DSS {stage} START" if args.action=="start" else f"DSS {stage} STOP"
+if not click.confirm(f"Are you sure you want to do this ({action})?"):
```

Comment:
How about mentioning the consequences here as in `The concurrency limit of each deployed lambda will be set to 0` for stop and `Any concurrency limits on deployed lambdas will be removed` for start.

Reply:
It seems reasonable to warn here.

Concurrency limits are lost during both actions btw.